galaxy_ng/app/access_control/statements/standalone.py

```diff
@@ -1,4 +1,17 @@
 STANDALONE_STATEMENTS = {
+    'AppRootViewSet': [
+        {
+            "action": ["list", "retrieve"],
```

Comment:
if this is only being used on the api root view, then you can set this to `"action": ["retrieve"]`